FraihmworkHealthAndIntegrityApi.yaml

```diff
@@ -1327,13 +1337,21 @@ components:
         - $ref: '#/components/schemas/Fault'
 
   securitySchemes:
+    ComponentAdmin:
```

Comment:
The new scopes / security schemes have not been propagated back to the original endpoints. Each endpoint above should be decorated with one or more of these identifiers to properly reflect the intended usage

Reply:
Still need to propagate FaultAdmin

Reply:
Done, thanks for the good catch :)